### PR TITLE
Campaign pages migration

### PR DESCRIPTION
**Pull request by dougdroper**
Migration from the old website campaigns to Contento.

### CODE REVIEW

File: app/views/campaigns/__campaign_template.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <%= heading_tag '', class: 'l-campaign__heading', level: 1 %>
+
+    </div>
+
+    <section class="l-campaign__next-steps">
+      <%= heading_tag '', class: 'l-campaign__next-steps-heading', level: 2 %>
+
+      <div class="next-step">
+        <a class="next-step__link" href="">
+          <%= heading_tag '', level: 2, class: 'next-step__heading' %>
```
Comment (tomas-stefano) [inline app/views/campaigns/__campaign_template.html.erb]: Is it normal the empty heading tags here?

File: app/views/campaigns/borrowing_get_the_facts.html.erb [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<div class="l-constrained">
+  <article>
+    <div class="l-campaign__content l-campaign__content--padded-top">
+      <%= heading_tag 'Borrowing - get the facts', class: 'l-campaign__heading', level: 1 %>
```
Comment (tomas-stefano) [inline app/views/campaigns/borrowing_get_the_facts.html.erb]: Can we move to a locale?

Comment (tomas-stefano) [inline app/views/campaigns/borrowing_get_the_facts.html.erb]: As I see more in the bottom, the campaigns not using locales. Is this page supposed to be on Contento :tm: ?

**Comment by tomas-stefano** [comment]
:+1:

**Comment by ourmaninamsterdam** [comment]
:+1: :ship:  LGTM

**Pull request merged**